src/containers/pagination/indexFunctioncalComponent.js

```diff
@@ -61,6 +61,9 @@ function PaginationProvider(props) {
           direction: props.direction,
         });
 
+      const data = findCurrentData(props.data, currentPage, selectedRowsPerPage);
```

Comment:
@harsh-syook should this be done for server-side-rendering also?
I'm guessing you'll be passing paginated data from the parent & this logic should be done only for the non-server-side-rendering. Please confirm

Reply:
@harsh-syook nudging